packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -0,0 +1,326 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BalanceTransferPayload, TransferDomainType } from '../../../src/category/account'
+import { RpcApiError } from '@defichain/jellyfish-api-core/dist/index'
+import BigNumber from 'bignumber.js'
+
+describe('TransferDomain', () => {
+  let dfiAddress: string, ethAddress: string
+  const amountToTransfer = 5
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+    await client.masternode.setGov({
+      ATTRIBUTES: {
+        'v0/params/feature/evm': 'true'
+      }
+    })
+    await container.generate(1)
+    dfiAddress = await container.call('getnewaddress')
+    // transfer utxostoaccount
+    await container.call('utxostoaccount', [{ [dfiAddress]: '100@0' }])
+    await createToken(dfiAddress, 'BTC', 200)
+    await container.generate(1)
+    ethAddress = await container.getNewAddress('', 'eth')
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  async function createToken (address: string, symbol: string, amount: number): Promise<void> {
+    const metadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+    await container.waitForWalletBalanceGTE(101)
+    await container.call('createtoken', [metadata])
+    await container.generate(1)
+
+    await container.call('utxostoaccount', [{ [address]: '100@0' }])
+    await container.generate(1)
+
+    await container.call('minttokens', [`${amount.toString()}@${symbol}`])
+    await container.generate(1)
+  }
+
+  it('should fail Transfer Domain from DFC to EVM if type is string', async () => {
```

Comment:
```suggestion
  it('should fail Transfer Domain from DVM Token to EVM if type is string', async () => {
```

You should keep the narrative consistent since the enum it's called `DVM Token` instead of `DFC`.